### PR TITLE
Update to 0.0.9.3

### PR DESCRIPTION
**Pull request by rushsteve1**
Updated to the latest release of 0.0.9.3
Also changed it to build from the release tarsand to use the external libqmatrixclient

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

File: com.github.quaternion.json [hunk 1]
```diff
@@ -14,15 +14,25 @@
         "--device=dri"
     ],
     "modules": [
+        {
+            "name": "libqmatrixclient",
```
Comment (TingPing) [inline com.github.quaternion.json:18]: This will install headers and such that need to be added to `cleanup`

Comment (rushsteve1) [reply com.github.quaternion.json:18]: Done in 669067c

**Comment by rushsteve1** [comment]
bot, build

File: com.github.quaternion.json [hunk 2]
```diff
@@ -14,15 +14,27 @@
...
+            "buildsystem": "cmake-ninja",
+	    "cleanup": ["/include", "/lib", "/share", "/bin/qmc-example"],
```
Comment (TingPing) [inline com.github.quaternion.json]:
```suggestion
            "cleanup": ["/include", "/lib", "/share", "/bin/qmc-example"],
```

**Comment by rushsteve1** [comment]
Thanks for the quick responses @TingPing!

**Pull request merged**